pkg/build/dockerfile-generator_test.go

```diff
@@ -46,98 +44,11 @@ func TestPorter_buildDockerfile(t *testing.T) {
 			gotDockerfile := strings.Join(gotlines, "\n")
 
 			wantDockerfilePath := fmt.Sprintf("testdata/%s.Dockerfile", driver)
-			wantDockerfile, err := ioutil.ReadFile(wantDockerfilePath)
-			require.NoError(t, err, "could not read %s", wantDockerfilePath)
-			assert.Equal(t, string(wantDockerfile), gotDockerfile)
+			test.CompareGoldenFile(t, wantDockerfilePath, gotDockerfile)
 		})
 	}
 }
 
-func TestPorter_buildDockerfile_alternateManifestLocation(t *testing.T) {
```

Comment:
I removed some tests that are no longer relevant or duplicate validations from other tests